paging/common/src/main/kotlin/androidx/paging/PageFetcher.kt

```diff
@@ -145,16 +141,17 @@ internal class PageFetcher<Key : Any, Value : Any>(
                 accessor.state.collect {
```

Comment:
Btw - now that we send the whole CombinedLoadStates, let's just send a single event instead of three here.

Reply:
Sounds good!